### PR TITLE
Issue 214

### PR DESCRIPTION
**Pull request by karenang**

### CODE REVIEW

File: app/scripts/help/keyboardShortcuts.json [hunk 1]
```diff
@@ -18,19 +18,39 @@
...
       {
-        "title": "Timetable",
-        "keys": ["T"],
-        "description": "Pressing <kbd>T</kbd> will quickly jump you to the [Timetable](/timetable) page."
+        "title": "Venues",
+        "keys": ["V"],
+        "description": "Pressing <kbd>V</kbd> will quickly jump you to the [Venues](/venues) page."
+      },
+      {
+        "title": "Jump to Venue Finder",
```
Comment (yangshun) [inline app/scripts/help/keyboardShortcuts.json]: Better if it's just **Venue Finder**

**Comment by yangshun** [comment]
Thanks @karenang . :joy:

**Comment by karenang** [comment]
Thanks! :v:

File: app/scripts/help/keyboardShortcuts.json (hunk 1 above)
Comment (karenang) [inline app/scripts/help/keyboardShortcuts.json]: :unamused:

**Pull request merged**